server/handlers.go

```diff
@@ -78,6 +78,7 @@ type discovery struct {
 	Keys              string   `json:"jwks_uri"`
 	UserInfo          string   `json:"userinfo_endpoint"`
 	DeviceEndpoint    string   `json:"device_authorization_endpoint"`
+	Introspect        string   `json:"introspect_endpoint"`
```

Comment:
I believe this should be `introspection_endpoint` as per RFC.

Reply:
Are you sure? Could you point to the RFC where it is described?  In Keycloak, for example, it is `token_introspection_endpoint`.

Reply:
I belive he refers to this RFC: https://datatracker.ietf.org/doc/html/rfc8414